app/api/v1/endpoints/media.py

```
@@ -584,42 +597,56 @@ async def get_media_thumbnail_signed(
-                return StreamingResponse(
-                    response.aiter_bytes(),
-                    media_type=response.headers.get("content-type", "image/jpeg"),
-                    headers={
-                        "Cache-Control": "public, max-age=3600",
-                        "X-Provider": "immich"
-                    },
-                    background=BackgroundTask(_close_httpx_stream, response)
-                )
+                # Session closed, but streaming response is still open
+            except Exception as e:
+                 error_logger.exception(f"Proxy thumbnail failed: {e}")
```

Ruff (0.14.13)

Redundant exception object included in `logging.exception` call

(TRY401)